src/HealthChecks.AzureServiceBus/AzureEventHubHealthCheck.cs

```diff
@@ -78,4 +81,14 @@ private string GetFullConnectionString()
             connectionString = $"{connectionString};{ENTITY_PATH_SEGMENT}{_options.EventHubName}";
         return connectionString;
     }
+
+    private static EventHubProducerClientOptions CreateClientOptions(HealthCheckContext context) => new()
+    {
+        ConnectionOptions = new()
+        {
+            ConnectionIdleTimeout = context.Registration.Timeout.TotalMilliseconds > 0
+                ? context.Registration.Timeout
+                : TimeSpan.FromSeconds(DEFAULT_TIME_OUT_IN_SECONDS), // Default in EventHub SDK
```

Comment:
I would not set default value. Just do not set value at all.

Reply:
Thanks for suggestion, it's been changed.